content/learn/05.communication/10.can/can.md

```diff
@@ -0,0 +1,243 @@
+
+- The CAN class provides methods to manage CAN communication typically used with specific Arduino boards that have CAN capability, like the [Arduino UNO R4](https://store.arduino.cc/products/arduino-due) or with add-on shields such as the MKR CAN Shield.
+
+The CAN class has several essential methods:
+
+- `begin(CanBitRate rate)` - Initializes the CAN bus with a specified bit rate (e.g., `CanBitRate::BR_250k` for 250 kbps).
+
+- `available()` - Checks if there are any incoming CAN messages available to read.
+
+- `read()` - Reads a CAN message from the bus and returns it as a `CanMsg` object.
+
+- `write(const CanMsg& msg)` - Writes a CAN message to the bus. The message is encapsulated in a `CanMsg` object, which includes the CAN ID, data length, and the data payload.
+
+These methods provide the core functionality for sending and receiving messages over the CAN bus, allowing efficient and reliable communication across multiple devices in a network. For example, these methods can be used to monitor vehicle sensors or control industrial machines.
+
+## Arduino CAN Pins
+
+The default pins on an Arduino board are CAN High and CAN Low. See table below for more information:
+
+***Depending on which Arduino board you are using, you will need a transceiver/receiver to read the differential signal properly. Check the product page of the board you are using for more information.***
+
+| Board / Shield | CAN0_RX   | CAN0_TX   | CAN1_RX | CAN1_TX |
+| -------------- | --------- | --------- | ------- | ------- |
+| UNO R4         | D13       | D10       |         |         |
+| DUE            | D68       | D69       | D66     | D53     |
+| MKR CAN Shield | dedicated | dedicated |         |         |
+
+## Technical Specifications
+
+### History of CAN
+
+The CAN bus, or Controller Area Network, was developed by the German company Bosch in the 1980s. Its primary purpose was to streamline communication within vehicles, enabling various microcontrollers and devices to communicate with each other without the need for a central host computer. This innovation significantly enhanced the efficiency and reliability of automotive systems, allowing for real-time data exchange and coordinated control across different vehicle components. The robustness and noise immunity of CAN made it particularly suitable for the harsh environments such as cars.
+
+| Name                        | Year                       | Speed                                                 |
+|-----------------------------|----------------------------|-------------------------------------------------------|
+| CAN 1.0                     | 1986                       | Up to 125 kilobits per second                         |
+| CAN 2.0                     | 1991                       | Up to 1 megabit per second                            |
```

Comment:
kilobit can be lowercase k as it is below 1 000 000 as far as I know Mega should always be capitalized https://en.wikipedia.org/wiki/Metric_prefix this is mostly irrelevant maybe so don't feel forced to adopt it :) If you do find relevant there are more instances on the document.
```suggestion
| CAN 2.0                     | 1991                       | Up to 1 Megabit per second                            |
```